### PR TITLE
Adding dashboard feature documentation

### PR DESCRIPTION
**Pull request by amuthanarulraj**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,3 +4,4 @@
 Feature 1: Login
 Feature 2: Logout
 Feature 3: Home Page
+Feature 4: Dashboard
```
Comment (amuthanarulraj) [inline README.md:7]: Can you explain feature?

Comment (amuthanarulraj) [reply README.md:7]: Its all fine.

Comment (amuthanarulraj) [reply README.md:7]: Its resolved

**Comment by Ashish310** [comment]
Looks good!

**Pull request merged**